sal-service/src/main/java/org/ow2/proactive/sal/service/util/ClusterUtils.java

```diff
@@ -61,7 +61,7 @@ public class ClusterUtils {
     private RepositoryService repositoryService;
```

Comment:
I know it's not from your PR but please remove this Autowired field if it's not used.

Reply:
done